demo/image/placeholder/playground

```diff
@@ -0,0 +1,34 @@
+const placeholder= (
+  <svg viewBox="0 0 1 1">
+      <rect width="100%" height="100%" />
+  </svg>
+)
+
+return (
+  <div>
+    <h2>Image Placeholder</h2>
+    <div style={{width: '20%'}}>
+      <ImagePlaceholder
+        image={{
+          src: 'https://satyr.io/1000?delay=3g',
+          alt: 'Some alt'
+        }}
+        placeholder={placeholder}
+      />
+    </div>
+    <h2>Fallback example</h2>
+    <div style={{width: '20%'}}>
+      <ImagePlaceholder
```

Comment:
I this a new SUI component from UX ?
I'm not a big fan of passing the image as properties. 
What about if my image has a title or other image properties that are not mention in ths shape you define ?
I'd rather imagine ImagePlaceholder as a enhence IMG tag and would do: 

```
<ImagePlaceholder
  src= 'http://404'
  alt= 'Some alt'
  placeholder={placeholder}
  fallbackImage={<img src="https://satyr.io/500" alt="" />}
/>
```
What do you think ?
I find weird that placeholder is a component but fallback an object.

Reply:
No, its for our logos component (without extra UX definition). I like your proposal